pubspec.lock

```diff
@@ -167,10 +167,10 @@ packages:
     dependency: "direct dev"
     description:
       name: flutter_lints
-      sha256: "9e8c3858111da373efc5aa341de011d9bd23e2c5c5e0c62bccf32438e192d7b1"
+      sha256: "3f41d009ba7172d5ff9be5f6e6e6abb4300e263aab8866d2a0842ed2a70f8f0c"
       url: "https://pub.dev"
     source: hosted
-    version: "3.0.2"
+    version: "4.0.0"
```

Comment:
We don't need changes in pubspec* files as dependabot will create PR's to update them. Can you remove both changes

Reply:
ok , I'm looking into it.